### PR TITLE
New layer config for shop swiss map vector 500 and 100

### PR DESCRIPTION
**Pull request by pfanguin**
Test link : https://mf-geoadmin3.dev.bgdi.ch/ltfap/?lang=de&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.swiss-map-vector1000.metadata,ch.swisstopo.swiss-map-vector500.metadata&wms_url=%2F%2Fwms-bgdi.dev.bgdi.ch%2Fltfap&X=183500.00&Y=681000.00&zoom=1

Must be on **int for 19.10.2016** and on **prod for 26.10.2016**

### CODE REVIEW

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -39,6 +39,34 @@ class DosisleistungTerrestrisch(Base, Vector):
...
 
+class smv1000(Base, Vector):
+    __tablename__ = 'smv1000'
+    __table_args__ = ({'schema': 'public', 'autoload': False})
+    __template__ = 'templates/htmlpopup/smv.mako'
+    __bodId__ = 'ch.swisstopo.swiss-map-vector1000.metadata'
+    id = Column('perimeter', Integer, primary_key=True)
+    scale = Column('n_scale', Integer)
+    price = Column('n_price', Integer)
+    release = Column('dt_release', Text)
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: same here

File: chsdi/models/vector/stopo.py [hunk 2]
```diff
@@ -39,6 +39,34 @@ class DosisleistungTerrestrisch(Base, Vector):
...
+    the_geom = Column(Geometry2D)
+
+register('ch.swisstopo.swiss-map-vector1000.metadata', smv1000)
+
+
+class smv500(Base, Vector):
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: Class name with a capital letter

**Comment by loicgasser** [comment]
Minor comments otherwise LGTM

File: chsdi/models/vector/stopo.py (hunk 1 above)
Comment (pfanguin) [inline chsdi/models/vector/stopo.py]: Yes of course sorry.

**Comment by pfanguin** [comment]
It's done, thanks.

**Comment by loicgasser** [comment]
jenkins test

File: chsdi/templates/htmlpopup/smv.mako [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <tr><td class="cell-left">${_('smv.scale')}</td>   <td>${c['attributes']['scale']}</td></tr>
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/smv.mako]: This means that you made sure scale was not nullable (which is good). so I would add nullable false in model and DB, or if not the case -> `${c['attributes']['scale'] or '-'} `

**Comment by loicgasser** [comment]
one last tiny comment and I merge.

**Comment by pfanguin** [comment]
It's done, thanks.

**Pull request merged**